src/models/m_category.erl

```diff
@@ -679,9 +676,7 @@ ensure_hierarchy(Context) ->
                     lager:warning("[~p] Ensure category found ~p new categories.", [z_context:site(Context), N]),
                     flush(Context);
                 {ok, 0} ->
-                    ok;
-                {error, _} = Error ->
-                    Error
+                    ok
             end;
         true ->
             lager:warning("[~p] Ensure category requested while renumbering.", [z_context:site(Context)]),
```

Comment:
Line 682 is too long:             lager:warning("[~p] Ensure category requested while renumbering.", [z_context:site(Context)]),.